web/components/layout/auth/Sidebar.tsx

```diff
@@ -104,9 +104,9 @@ const Sidebar = ({ changelog, setOpen, sidebarRef }: SidebarProps) => {
           },
           {
             name: "Playground",
-            href: "/prompts/fromPlayground",
+            href: "/playground",
             icon: TestTube2,
-            current: pathname.includes("/prompts/fromPlayground"),
+            current: pathname.includes("/playground"),
```

Comment:
probably happened because of the merge, we should not route to playground for now - it's still behind a "feature flag"
revert back to `/prompts/fromPlayground`